docs/detections/query-alert-indices.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[query-alert-indices]]
+== Query alert indices
+This page explains how you should query alert indices, for example when building rule queries, custom dashboards, or visualizations. For more information about alert event field definitions, review the [Alert schema].
+
+[discrete]
+=== Alert indices
+Alert events are stored in hidden {es} indices. The naming conventions for these indices and their aliases differ depending on which version of {stack} you're running:
+
+* *8.0.0 and later:* `.internal.alerts-security.alerts-<space-id>-NNNNNN`, where `NNNNNN` is a number that increases over time, starting from 000001.
+
+* *Before 8.0.0:* `.siem-signals-<space-id>-NNNNNN`, where `NNNNNN` is a number that increases over time, starting from 000001.
+
+[discrete]
+=== Alert index aliases
+We recommend querying the following index aliases to avoid querying the numbered alert indices:
+
+* If you're using version 8.0.0 or later: `.alerts-security.alerts-<space-id>`
+This alias includes both the legacy (before 8.0.0) and the new alert indices.
+
+* If you're using a version before 8.0.0: `.siem-signals-<space-id>`
+Queries on this alias will function regardless of your {stack} version, but will not follow the newer `.alerts` naming convention, and may be deprecated in future releases.
```

Comment:
```suggestion
Queries on this alias will function regardless of your {stack} version, but will not follow the newer `.alerts` naming convention and may be deprecated in future releases.
```

Reply:
The first comma ("version, but") is also grammatically incorrect, but prob OK to keep if you feel a comma pause is warranted